packages/create-cloudflare/README.md

```diff
@@ -10,7 +16,44 @@ pnpm create cloudflare@latest
 yarn create cloudflare
 ```
 
+### Setup via CLI Arguments
+
+#### New Websites or Web applications via Frameworks
+
+To create a new website or web framework without interaction, run:
+
+```bash
+npm create cloudflare@latest -- --type webFramework -- --framework <frameworkName>
```

Comment:
There are two `--` markers here. Is that correct?

Reply:
Good catch. I removed those.